src/components/AppLayout/TopNav/TopNavLink.tsx

```diff
@@ -0,0 +1,17 @@
+import { ArrowLeftIcon, Button } from "@saleor/macaw-ui/next";
+import React from "react";
+import { Link } from "react-router-dom";
+
+export const TopNavLink: React.FC<{
+  to: string;
+  variant?: "secondary" | "tertiary";
```

Comment:
Should it really allow variants to be choose from?

Reply:
I think yes - I wanted to use the same component in all places and variant prop is only thing that changes for now :)

Reply:
Ok, I thought its App specific, because `AppLayout` confused me (maybe you should rethink the name)

Reply:
We want to do this - in the next PR

Reply:
We want to do this - in the next PR 😄